rslib/src/notetype/render.rs

```diff
@@ -82,7 +82,7 @@ impl Collection {
         })
     }
 
-    fn render_card_inner(
+    pub fn render_card_inner(
```

Comment:
_inner routines are not intended to be public - I'd either make it pub(crate) or rename it